doc/specs/stdlib_linalg.md

```diff
@@ -898,3 +900,176 @@ Exceptions trigger an `error stop`.
 ```fortran
 {!example/linalg/example_determinant2.f90!}
 ```
+
+## `eig` - Eigenvalues and Eigenvectors of a Square Matrix
+
+### Status
+
+Experimental
+
+### Description
+
+This subroutine computes the solution to the eigenproblem \( A \cdot \bar{v} - \lambda \cdot \bar{v} \), where \( A \) is a square, full-rank, `real` or `complex` matrix.
+
+Result array `lambda` returns the eigenvalues of \( A \). The user can request eigenvectors to be returned: 
+on output `left` may contain the left eigenvectors, `right` the right eigenvectors of \( A \). 
```

Comment:
This sentence is unclear. Is it something missing after "returned:"?

Reply:
Thank you @jvdp1: I've refactored it here [f13e75d](https://github.com/fortran-lang/stdlib/pull/816/commits/f13e75d76f12af4c2084efc0b962d05b1474ebc2)